src/test/java/org/folio/service/claiming/ClaimingServiceTest.java

```diff
@@ -0,0 +1,10 @@
+package org.folio.service.claiming;
+
+import org.junit.jupiter.api.Test;
+
+class ClaimingServiceTest {
+
+  @Test
+  void sendClaims() {
+  }
```

Comment:
If no need to test the service, can be removed as coverage it enough, but can also use copilot to quickly generate some basic tests

Reply:
> If no need to test the service, can be removed as coverage it enough, but can also use copilot to quickly generate some basic tests

Yeah, I was planning today, if I manage to squeeze some from Copilot I will push them or will remove this class entirely